packages/sui-studio/src/components/layout/Logo.js

```diff
@@ -1,4 +1,4 @@
-import {getStudioLogo} from '../utils'
+import {getStudioLogo} from '@s-ui/studio/src/components/utils'
```

Comment:
It could be better to use relative import instead, as it was

Reply:
yes! I agree 
don't know what happened here

Reply:
✅ done